ray-operator/controllers/ray/common/service_test.go

```diff
@@ -78,4 +78,29 @@ func TestBuildServiceForHeadPod(t *testing.T) {
 	if !reflect.DeepEqual(expectedResult, actualResult) {
 		t.Fatalf("Expected `%v` but got `%v`", expectedResult, actualResult)
 	}
+
+	actualResult = svc.Spec.Selector[KubernetesApplicationNameLabelKey]
+	expectedResult = ApplicationName
+	if !reflect.DeepEqual(expectedResult, actualResult) {
+		t.Fatalf("Expected `%v` but got `%v`", expectedResult, actualResult)
+	}
+}
+
+func TestBuildServiceForHeadPodWithDefaultLabel(t *testing.T) {
```

Comment:
Maybe something like `TestBuildServiceForHeadPodWithAppNameLabel`, since it's testing using a non-default ApplicationName Label?

Reply:
Updated.